cli/config/configfile/file_test.go

```diff
@@ -413,3 +415,13 @@ func TestCheckKubernetesConfigurationRaiseAnErrorOnInvalidValue(t *testing.T) {
 		}
 	}
 }
+
+func TestSave(t *testing.T) {
+	configFile := New("test-save")
+	defer os.Remove("test-save")
+	err := configFile.Save()
+	assert.NilError(t, err)
+	cfg, err := ioutil.ReadFile("test-save")
+	assert.NilError(t, err)
+	assert.Check(t, is.Equal(string(cfg), "{\n	\"auths\": {}\n}"))
```

Comment:
Feels like an omitempty tag is missing in the `Auths` field.

Reply:
We can fix in another PR

Reply:
nit: Review after merge (so maybe for the just opened follow-up) 😅:
You can use backquotes instead:
```go
assert.Check(t, is.Equal(string(cfg), `{
	"auths": {}
}`))
```